tests/longevity-tests/run-longevity.bash

```diff
@@ -47,10 +47,11 @@ if [[ $1 != "6.0" && $1 != "6.5" && $1 != "6.7" ]]; then
     exit 1
 fi
 
-if [[ ! $(grep dns /etc/docker/daemon.json) ]]; then
+if [[ ! $(grep dns /etc/docker/daemon.json) && ! $(grep insecure-registries /etc/docker/daemon.json) ]]; then
```

Comment:
If both lines (insecure-registries and dns) are necessary, shouldn't the `&&` on this line be `||`?